### DISCUSSION TITLE
A (virtual) Zim Conference

### DISCUSSION THREAD

**Discussion by shivams** [zim-desktop-wiki/zim-desktop-wiki#2100] [Polls]
I have been thinking of pitching this idea for a while. We should have a ZimCon (a virtual one)! Having been a Zim user for many years, trying out many other solutions, and coming back to Zim every time, I'd like to know how other people organize their knowledge and life in Zim.  I'd especially be interested in seeing how "heavy" organizers use their notebooks. We can have show-and-tell sessions. We can discuss the ideas we'd like to see incorporated into Zim. 

I'm sure there would be quite some people interested in such a conference. Many other knowledge management tools have been aggressive towards such activities within their communities (often owing to commercial motivation). Zim should organize such conferences too. It may help shape the development of Zim itself!

**Comment by dehesselle** [comment]
Nice idea!

Something to consider: if you want to reduce the organizational burden, you could start with organizing a Zim booth at some general Linux or FOSS conference before creating your own conference. (This is something I see other projects do.)

**Reply by shivams** [reply]
Aaah. That's a very good suggestion. I will see if I can find a local FOSS conference; maybe start with an event at my university itself. I doubt there will be many Zim fans in my university here (some use org-mode, many use Evernote or Notion, and most use Google Docs to organize their knowledge), but yeah I can have a "booth" for Zim. Will see. I'll pitch to the grad students community here.

Meanwhile, we can then have an online "meetup" for Zim (instead of a conference). And actually, a "meetup" is really more what I had in mind. A "meetup" for "showing-and-telling" about how you organize, how would you like your setup to be etc. A couple-hour event at best. And depending on the turnout and request, we can plan something later.

**Reply by rolfkleef** [reply]
I'd say FOSDEM is a good candidate, with several thousands of FOSS developers, the first weekend in February, in Brussels and online. A lightning talk or session should be feasible to work as a "Birds of a feather" gathering. https://fosdem.org

**Comment by jaap-karssenberg** [comment]
For reference, material from previous presentations - all a few years old,
so will need updating:

* By Brendan: Getting Work Done in Zim [Glump.net]
<https://www.glump.net/howto/desktop/getting-work-done-in-zim> (2013, for
the Boston Linux user group)
* By Jaap: Index of /presentations (zim-wiki.org)
<https://zim-wiki.org/presentations/> (2015, for the Dutch Linux user
group, English version available as well)

**Comment by mikro2nd** [comment]
If a full-blown conference sounds too daunting to organise, perhaps consider something more lightweight -- a once-a-month online get-together, perhaps.
